### ISSUE TITLE
NVDA does not announce when application volume control feature is toggled using assigned gesture

### ISSUE THREAD

**Issue by HBM301** [nvaccess/nvda#17882]
### Steps to reproduce:

- Go to NVDA menu, Preferences and Input gestures

- assign a gesture  to turn on or off  application volume control feature, for example: NVDA+alt+V (all layouts)

### Actual behavior:

try press NVDA+alt+V, NVDA does not announce that this feature has been turned on or off, when entering the gesture has been assigned, there is an error

### Expected behavior:

NVDA should notify the user when they turn on or off this feature, for example: Volume control of the application  enable or something

### NVDA logs, crash dumps and other attachments:
ERROR - scriptHandler.executeScript (12:59:52.872) - MainThread (4624):
error executing script: <bound method GlobalCommands.script_toggleApplicationsVolumeAdjuster of <globalCommands.GlobalCommands object at 0x066C87D0>> with gesture 'NVDA+alt+v'
Traceback (most recent call last):
  File "scriptHandler.pyc", line 300, in executeScript
  File "globalCommands.pyc", line 4908, in script_toggleApplicationsVolumeAdjuster
  File "audio\appsVolume.pyc", line 151, in _toggleAppsVolumeState
  File "audio\appsVolume.pyc", line 90, in _updateAppsVolumeImpl
AttributeError: 'AppsVolumeAdjusterFlag' object has no attribute 'calculated'

#### NVDA installed/portable/running from source:
Portable (Latest Alpha)
#### NVDA version:
35820,0fc7254e (2025.1.0.35820)
#### Windows version:
Windows 11 24H2 (AMD64) build 26100.3624
#### Does the issue still occur after restarting your computer?
Yes 
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No, because the previous versions of NVDA  (2024.4.2) don't  have this feature
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Not tested
